lib/logstash/plugin_mixins/rabbitmq_connection.rb

```diff
@@ -48,6 +48,12 @@ def setup_rabbitmq_connection_config
         # Time in seconds to wait before retrying a connection
         config :connect_retry_interval, :validate => :number, :default => 1
 
+        # The default connection timeout; zero means wait indefinitely
+        config :connection_timeout, :validate => :number
+
+        # Heartbeat delay in seconds. Default is 0 (no heartbeats)
+        config :heartbeat, :validate => :number, :default => 0
```

Comment:
Hmm, I got that straight from the march_hare API, but you're right. I'll override that